src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/xfpage/_cq_dialog/.content.xml

```diff
@@ -95,6 +95,29 @@
                                                 targetProperty="jcr:content/cq:products"/>
                                         </items>
                                     </productsPickerSection>
+                                    <categoriesPickerSection
```

Comment:
I think this kind of dialog extension would need to go in an xfpage extension in the CIF component project. Then both the Venia xfpage and this xfpage will inherit it, there will be no duplication and it gets also delivered to existing customers (not using Venia or archetype) when they do an upgrade.

Reply:
I agree with @LSantha but I think we can merge this, create the new `xfpage` component + dialog in CIF, release CIF, and then update the archetype. @LSantha did you create an issue to track this and discuss with the team?

Reply:
I've created and marked it for tech grooming.